crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -2862,6 +2862,7 @@ where
                     Rule::StartProcessWithAShell,
                     Rule::StartProcessWithNoShell,
                     Rule::StartProcessWithPartialPath,
+                    Rule::UnixCommandWildcardInjection,
```

Comment:
"linux" in [bandit's`linux_commands_wildcard_injection` plugin name](https://bandit.readthedocs.io/en/latest/plugins/b609_linux_commands_wildcard_injection.html#module-bandit.plugins.injection_wildcard) is a misnomer as this applies to other unixy os's too. Therefore I chose to "fix" this in the name, but I don't know how truthfully we should follow upstream names here.